lib/server_health_check_rails/logger.rb

```diff
@@ -0,0 +1,63 @@
+# The health_index_path route is pinged frequently. We don't want that to clutter
+# our main log file, so we intercept all calls to that route and change the logger.
+module ServerHealthCheckRails
```

Comment:
I started writing this code for my application, and then I realized that it's really not specific to my application, but to any user of this gem.  Our devops team pings this route constantly, which is what motivated this change, and that is going to be the case for all of our applications.